app/components/market/submit/parameter-input.component.ts

```diff
@@ -45,7 +52,35 @@ export class ParameterInputComponent implements ControlValueAccessor, OnDestroy
     }
 
     public validate() {
-        return null;
+        const valid = this.parameterValue.valid;
+        if (valid) {
+            return null;
+        } else {
+            let messageText = "unknown error";
+            if (this.parameterValue.errors.minlength) {
+                messageText = "FormControl minLength error:";
+                messageText = messageText + " minlength: " +  this.parameterValue.errors.minlength.requiredLength;
```

Comment:
Save ` this.parameterValue.errors` in a local variable so you don't have to repeat all that.
Also make the message look nicer like "Should be at least x characters`

Reply:
And try to use string interpolation `${}` instead of `+`